### PR TITLE
fix: dashboard won't try to load datasets and projects for AnonymousUser

### PR DESCRIPTION
**Pull request by jLebioda**

### CODE REVIEW

**Comment by jLebioda** [comment]
closes #351

File: web/views/dashboard.py [hunk 1]
```diff
@@ -20,15 +23,26 @@ def dashboard(request):
...
-    # Get the notifications
-    # notifications = request.user.notifications.ordered()[:5]
+    # Return lists instead of QuerySets
+    return list(last_datasets), list(last_projects)
+
 
+def dashboard(request: HttpRequest) -> HttpResponse:
+    the_user = request.user
+
+    if the_user.is_anonymous():
```
Comment (vildead) [inline web/views/dashboard.py:33]: is_anonymous is not a method but attribute 

https://docs.djangoproject.com/en/4.0/ref/contrib/auth/#django.contrib.auth.models.User.is_anonymous

**Review by vildead — CHANGES_REQUESTED** [review]
With a small fix I get no error.

 Unfortunately, the `login` page is just skipped and I can access all records without any authentication ☹️

File: web/views/dashboard.py (hunk 1 above)
Comment (jLebioda) [reply web/views/dashboard.py:33]: my bad, I got misled by VSCode's hint; thanks!

**Comment by jLebioda** [comment]
> With a small fix I get no error.
> 
> Unfortunately, the `login` page is just skipped and I can access all records without any authentication ☹️

It's actually interesting what happened with the authentication and why AnonymousUser is making his way in. I'll try to see what went wrong

**Comment by vildead** [comment]
I cannot reproduce the behaviour on current `develop` (https://github.com/elixir-luxembourg/daisy/commit/95a0c1d7d90d4deffaf350e418895517ea258e33`)

Maybe we dont need to merge at all?

**Comment by neoflex** [comment]
Django-stronghold doesn't work with Django 3.x if DEBUG mode is on and MEDIA_URL is not set in settings (can be set to None to fix it).
See https://github.com/mgrouchy/django-stronghold/issues/82
As Django-stronghold doesn't seem to be really maintained anymore, we could replace with https://pypi.org/project/django-require-login/.

**Comment by vildead** [comment]
It will be addressed when solving issue https://github.com/elixir-luxembourg/daisy/issues/378
Now closing.

**Pull request closed without merging**